### PR TITLE
Added line that selects the "More modules..." after adding a step.

### LINKED ISSUE #697: Select Module Does Not Reset After Adding A Module
### Please describe the problem (or idea)
The `select` dropdown for adding modules doesn't reset to its normal text. i.e. `More Modules...` after adding a module. The name of the module is shown but the description is not shown above.

> What happened just before the problem occurred? Or what problem could this idea solve?

screenshot from 2019-01-22 11-17-00

> What did you expect to see that you didn't?

screenshot from 2019-01-22 11-18-21

### FTO Candidate

This is a potential `first-timers-only` issue. If you are new to the community, then you can reformat this issue into a first timer only issue and mention `@publiclab/is-reviewers` to add the label. If you have contributed to Image Sequencer before, then you can resolve this one as a second time contribution.

#### Code to look into

https://github.com/publiclab/image-sequencer/blob/17d79c4b44c5e5b8f7f8b4574a783d5b87cdf871/examples/lib/intermediateHtmlStepUi.js#L53-L57

****

## Thank you!

Your help makes Public Lab better! We *deeply* appreciate your helping refine and improve this site.

To learn how to write really great issues, which increases the chances they'll be resolved, see:

https://publiclab.org/wiki/developers#Contributing+for+non-coders

### PR DESCRIPTION
**Pull request by gfting**
Fixes #697 

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `npm test`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [x] ask `@publiclab/reviewers` and `@publiclab/is-reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

Thanks!

### CODE REVIEW

**Comment by gfting** [comment]
@publiclab/is-reviewers thanks!

**Comment by HarshKhandeparkar** [comment]
@gfting you don't have to add the green tick in the checkboxes. They can be interacted with

**Comment by HarshKhandeparkar** [comment]
@publiclab/is-reviewers

**Comment by HarshKhandeparkar** [comment]
Please share a GIF. Thanks!

**Comment by HarshKhandeparkar** [comment]
Please share a GIF of this change in action.

**Comment by gfting** [comment]
@HarshKhandeparkar here's the GIF and the fix! Thanks!
imagesequencerfix

**Comment by aashna27** [comment]
> Please share a GIF of this change in action.
I guess it's okay , I have tested it locally it works. 🙌 
@gfting thanks a lot for the great work, and @HarshKhandeparkar you are awesome with your support.

**Comment by HarshKhandeparkar** [comment]
@gfting awesome.

**Comment by gfting** [comment]
@HarshKhandeparkar your support was seriously incredible during this - thank you very much!

File: dist/image-sequencer-ui.js [hunk 1]
```diff
@@ -79,6 +79,7 @@ window.onload = function() {
     ui.selectNewStepUi();
     ui.addStepUi();
     $(this).removeClass('selected');
+    // ui.removeStepUi();
```
Comment (rexagod) [inline dist/image-sequencer-ui.js]: Just one question. What does this comment denote and is it required?

Comment (gfting) [reply dist/image-sequencer-ui.js]: My apologies, that was a mistake and thought that it was removed

**Comment by gfting** [comment]
@rexagod I'm not sure what's going on with this branch and conflicts - what should I do? Thanks!

**Comment by HarshKhandeparkar** [comment]
@gfting, please resolve the conflicts by running `git merge upstream/main`
locally. This will add conflict markers to the confliction files. The
marker will marker what is the upstream change(i.e. the file in the
publiclab repo) and the current change(i.e. condition of the file in your
PR). The file has changed in the publiclab repo since some PRs have been
merged ever since you opened your PR. All you have to do is open all of the
source files(not the dist files) and combine both the upstream and current
changes in such a way that both the features work. Then you can run `grunt
build` which will resolve the dist file conflicts. Then you can commit the
changes. You might have to force push the commit. You can ask for help if
you are stuck.

On 24-Jan-2019 9:05 AM, "Gabriel Ting" <notifications@github.com> wrote:

@rexagod <https://github.com/rexagod> I'm not sure what's going on with
this branch and conflicts - what should I do? Thanks!

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/publiclab/image-sequencer/pull/706#issuecomment-457054885>,
or mute the thread

**Comment by HarshKhandeparkar** [comment]
This link might be helpful.

https://help.github.com/articles/resolving-a-merge-conflict-using-the-command-line/

**Comment by HarshKhandeparkar** [comment]
@gfting you will have to add the remote
`git remote add upstream https://github.com/publiclab/image-sequencer.git`

and then merge `git merge upstream/main`

The `/main` is necessary.

**Comment by gfting** [comment]
@HarshKhandeparkar I followed that, but still got the same error message

**Comment by HarshKhandeparkar** [comment]
@gfting did you merge `upstream/master` or `jywarren/main` ? Your PR seems to be a a lot of commits behind master. It seems like instead of updating your branch, you did something else. Please try to fix that by merging the proper upstream. You can ask for help here.

**Comment by gfting** [comment]
I'm not sure what happened here - I merged upstream master

**Comment by HarshKhandeparkar** [comment]
@gfting master is not the branch you need to merge, the default branch for image sequencer is `main`

**Comment by HarshKhandeparkar** [comment]
Master is 100s of commits behind main

**Comment by HarshKhandeparkar** [comment]
What you can do is 
- copy your changes and save them in some local file 
- delete your local repo
- delete the fork
- fork again
- clone the fork
- run `npm run setup`
- create a new branch
- copy your changes
- close this PR for now
- open a new one later on 

This is why you should always create a new branch for doing such things so that you can fallback to the main brach in case such things happen. Sorry you have to take a lot of trouble to do this.

**Comment by gfting** [comment]
It says that I'm already up to date with main

**Comment by HarshKhandeparkar** [comment]
@gfting I dont know why it says that. Did you setup the remote correctly? In any case though, you will have to close this one I guess.

Or you can revert the last few commits

**Pull request closed without merging**